etc/calls/testcall.py

```diff
@@ -1,19 +1,20 @@
-#!/usr/bin/python -u 
+#!/usr/bin/env -S python3 -u 
 
-import cPickle
+import pickle
 import sys
-import urllib
+import urllib.parse
 import base64
 
 #
 # Output the maximum number of instances of this 'callable' to spawn
 # The absolute maximum is set in the configuration file via 'warpscript.call.maxcapacity'
 #
 
-print 10
+print(1)
 
 #
-# An example of Warpscript to trigger this script: NOW ISO8601 ->PICKLE ->B64URL 'testcall.py' CALL
+# An example of Warpscript to test this script: 
+# [ NOW ISO8601 ] ->PICKLE ->B64URL 'testcall.py' CALL B64-> PICKLE->
```

Comment:
Why change the example?

Reply:
The previous example did not show a fully pickle unpickle that manipulates LIST and MAP.